README.md

```diff
@@ -215,3 +215,21 @@ Ensure that the directory is created and owned by the host user.
 mkdir -p ~/.${REPO}/ccache
 sudo chown -R $(id -u):$(id -g) ~/.${REPO}/ccache
 ```
+
+# Additional Information
+## CppCon 2023 Slides
+
+The code in `functional_programming_tests` was actually presented at CppCon 2023 and ROSCon 2023. The abstract of the presentation is: 
+
+```
+In the field of robotics, Robot Operating System (ROS) is the de facto middleware of choice across academia and industry.
+As most code examples in ROS utilize an object-oriented approach, challenges can arise when writing tests for production code due to the boilerplate code ROS introduces, often leading to unexpected bugs or flaky tests.
```

Comment:
> due to the boilerplate

It's not really about the boilerplate from ROS. What are you trying to say here?

Reply:
> due to the network calls

I guess that's what it should be?

Reply:
Well, I guess that sentence doesn't make complete sense now... 🤔 I'll go ruminate on it.

Reply:
So, that replacement would make sense, but overall the sentence reads
- ros code is object oriented
- presumably you're going to mention why this isn't preferred
- because something is bad with testing it
- but, it's actually the network calls?
- and that makes tests flaky? what do network calls have to do with OO?